integration_tests/pages/plan-overview.ts

```diff
@@ -31,4 +31,8 @@ export default class PlanOverview {
   clickDownOnSummaryCard = (value: number) => {
     this.getSummaryCard(value).contains('.govuk-button', 'Move goal down').click()
   }
+
+  isGoalAchievedRadio = (value: string) => {
```

Comment:
presumably this should be in a new remove-goal.ts file rather than plan-overview?

Reply:
Agreed, it definitely should. Moved it now 👍

Reply:
oops, i was looking at Aiden's remove goal work when reviewing this, I did indeed mean achieve goal!